ui/shared/components/panel/variants/VariantGene.jsx

```diff
@@ -592,12 +618,15 @@ BaseVariantGene.propTypes = {
   showInlineDetails: PropTypes.bool,
   compoundHetToggle: PropTypes.func,
   hasRnaTpmData: PropTypes.bool,
-  rnaSeqData: PropTypes.object,
+  sampleGeneData: PropTypes.object,
 }
 
 const getRnaSeqProps = (state, ownProps) => ({
   hasRnaTpmData: getFamiliesByGuid(state)[ownProps.variant.familyGuids[0]]?.hasRnaTpmData,
-  rnaSeqData: getRnaSeqOutilerDataByFamilyGene(state)[ownProps.variant.familyGuids[0]],
+  sampleGeneData: {
```

Comment:
this should not be done in the UI. You should change the server side code to return `sampleGeneData` instead of rnaSeqData and phePriData

Reply:
no, we need it grouped by individual for the family page `RnaSeqOutliers` detail. You should just change the behavior of the `getRnaSeqOutilerDataByFamilyGene` selector to instead be `getScoresDataByFamilyGene` and have it include both rna seq and prioritization data. Thinking about it more, its probably better to leave the sever side changes you made as is and only update this one selector